experimental/auth-logic/unix_epoch_time_wrapper_test.go

```diff
@@ -42,20 +41,15 @@ func TestUnixEpochTimeWrapper(t *testing.T) {
 		}
```

Comment:
Is there a reason for having both Fatalf and panic here?

Reply:
this already got removed